models/public/cocosnet/model.yml

```diff
@@ -97,11 +97,11 @@ postprocessing:
     file: model_files/models/networks/correspondence.py
     pattern: '/ self.opt.down\)'
     replacement: '// self.opt.down)'
-  # MO can only convert Squeeze with fixed dimensions
+  # Replace Squeeze with Gather operation for support PyTorch >= 1.7
```

Comment:
I don't think you should just replace the comment like that; the original comment explains _why_ Squeeze is bad in this case.